docs/src/main/asciidoc/verifier_messaging.adoc

```diff
@@ -151,12 +154,13 @@ And the following Spock test would be created:
 include::{verifier_core_path}/src/test/groovy/org/springframework/cloud/contract/verifier/builder/MessagingMethodBodyBuilderSpec.groovy[tags=trigger_no_output_spock]
 ----
 
-=== Consumer Stub Side generation
+=== Consumer Stub Generation
 
-Unlike the HTTP part - in Messaging we need to publish the Groovy DSL inside the JAR with a stub. Then it's parsed on the consumer side
-and proper stubbed routes are created.
+Unlike the HTTP part, in messaging, we need to publish the Groovy DSL inside the JAR with
+a stub. Then it is parsed on the consumer side and proper stubbed routes are created.
 
-For more information please consult the Stub Runner Messaging sections.
+For more information, please consult the Stub Runner Messaging sections.
+// TODO Add a link to those sections.
```

Comment:
https://cloud.spring.io/spring-cloud-contract/spring-cloud-contract.html/#stub-runner-for-messaging

Reply:
Thank you. I'll add that.